src/main/java/org/checkerframework/checker/returnsrcvr/ReturnsRcvrAnnotatedTypeFactory.java

```diff
@@ -70,30 +67,27 @@ public Void visitExecutable(AnnotatedTypeMirror.AnnotatedExecutableType t, Void
     /**
      * Checks if the given element is a setter in either an AutoValue builder or a Lombok builder
      */
-    private boolean isBuilderSetter(ExecutableElement element) {
-      MethodTree methodTree = (MethodTree) declarationFromElement(element);
-      if (methodTree == null) {
-        return false;
-      }
+    private boolean isBuilderSetter(AnnotatedTypeMirror.AnnotatedExecutableType t) {
+      ExecutableElement element = t.getElement();
 
-      ClassTree enclosingType = TreeUtils.enclosingClass(getPath(methodTree));
-
-      if (enclosingType == null) {
+      // skip constructors
+      if (element.getKind().equals(ElementKind.CONSTRUCTOR)) {
         return false;
       }
 
-      boolean inAutoValueBuilder = hasAnnotation(enclosingType, AutoValue.Builder.class);
+      Element enclosingElement = element.getEnclosingElement();
+
+      boolean inAutoValueBuilder = hasAnnotation(enclosingElement, AutoValue.Builder.class);
       boolean inLombokBuilder =
-          hasAnnotation(enclosingType, lombok.Generated.class)
-              && enclosingType.getSimpleName().toString().endsWith("Builder");
+          hasAnnotation(enclosingElement, lombok.Generated.class)
+              && enclosingElement.getSimpleName().toString().endsWith("Builder");
 
       // if we are in an AutoValue Builder, this will be the element for the abstract Builder class
-      Element builderClassElem = TreeUtils.elementFromTree(enclosingType);
+      Element builderClassElem = enclosingElement;
```

Comment:
This local variable seems silly now, since we already have `enclosingElement` in scope. Why not just replace all the uses of this variable with `enclosingElement`?